ui/src/app/Deployment/DeploymentProgress/DeploymentProgress.tsx

```diff
@@ -21,7 +20,7 @@ export const DeploymentProgress = ({ progressData, stepsData }: IDeploymentProgr
   )
 
   // render progress bar only if no failed steps
-  const progressBar = (progressData.failedStepIds.length < 0 ?
+  const progressBar = (progressData.failedStepIds.length <= 0 ?
```

Comment:
The length of an array will never be smaller than zero, hence the condition with `===` will suffice.

Reply:
complete